src/definitions/transform.ts

```diff
@@ -54,13 +54,15 @@ function _getDef(): CodeKeywordDefinition {
 
       function transformExpr(ts: string[]): Code {
         if (!ts.length) return data
-        const t = ts.pop() as string
```

Comment:
I suggest to pass tNames.slice() above rather than to copy on every iteration... Mostly because to is clearer what is going on.

Reply:
Done 🙂